app/services/encrypted_email.rb

```diff
@@ -30,7 +30,7 @@ def decrypted
   end
 
   def fingerprint
-    Pii::Fingerprinter.fingerprint(email)
+    Pii::Fingerprinter.fingerprint(email.downcase)
```

Comment:
I was fixing an inconsistency with how the User model treats emails when fingerprinting:
https://github.com/18F/identity-idp/blob/master/app/models/concerns/user_encrypted_email_overrides.rb#L22

Yes, email is downcased consistently elsewhere.